pkg/components/aws-ebs-csi-driver/component_test.go

```diff
@@ -137,3 +139,133 @@ func TestStorageClassDisabled(t *testing.T) {
+	//nolint:lll
+	testCases := []struct {
+		name                 string
+		inputConfig          string
+		expectedManifestName string
+		expected             string
+		jsonPath             string
+	}{
+		{
+			name:                 "no_tolerations_node",
+			inputConfig:          `component "aws-ebs-csi-driver" {}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/node.yaml",
+			jsonPath:             "{.spec.template.spec.tolerations[0].operator}",
+			expected:             "Exists",
+		},
+		{
+			name: "tolerations_node",
+			inputConfig: `component "aws-ebs-csi-driver" {
+							tolerations {
+								key      = "lokomotive.io"
+								operator = "Equal"
+								value    = "awesome"
+								effect   = "NoSchedule"
+							}
+						}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/node.yaml",
+			jsonPath:             "{.spec.template.spec.tolerations[0].value}",
+			expected:             "awesome",
+		},
+		{
+			name:                 "no_tolerations_csi_controller",
+			inputConfig:          `component "aws-ebs-csi-driver" {}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/controller.yaml",
+			jsonPath:             "{.spec.template.spec.tolerations[0].operator}",
+			expected:             "Exists",
+		},
+		{
+			name: "tolerations_csi_controller",
+			inputConfig: `component "aws-ebs-csi-driver" {
+							tolerations {
+								key      = "lokomotive.io"
+								operator = "Equal"
+								value    = "awesome"
+								effect   = "NoSchedule"
+							}
+						}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/controller.yaml",
+			jsonPath:             "{.spec.template.spec.tolerations[0].key}",
+			expected:             "lokomotive.io",
+		},
+		{
+			name:                 "no_tolerations_snapshot_controller",
+			inputConfig:          `component "aws-ebs-csi-driver" {}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/statefulset.yaml",
+			jsonPath:             "{.spec.template.spec.tolerations[0].operator}",
+			expected:             "Exists",
+		},
+		{
+			name: "tolerations_snapshot_controller",
+			inputConfig: `component "aws-ebs-csi-driver" {
+							tolerations {
+								key      = "lokomotive.io"
+								operator = "Equal"
+								value    = "awesome"
+								effect   = "NoSchedule"
+							}
+						}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/statefulset.yaml",
+			jsonPath:             "{.spec.template.spec.tolerations[0].effect}",
+			expected:             "NoSchedule",
+		},
+		{
+			name: "affinity_csi_controller",
+			inputConfig: `component "aws-ebs-csi-driver" {
+							node_affinity {
+								key      = "lokomotive.io/role"
+								operator = "In"
+								values   = ["storage"]
+							}
+						}`,
+			expectedManifestName: "aws-ebs-csi-driver/templates/controller.yaml",
+			jsonPath:             "{.spec.template.spec.affinity.nodeAffinity.requiredDuringSchedulingIgnoredDuringExecution.nodeSelectorTerms[0].matchExpressions[0].key}",
```

Comment:
You can break the string with `+` operator, so it's not that long. Then we can keep `lll` linter.